### DISCUSSION TITLE
Anyone else interested in Terraformer for harvesting PagerDuty Event Orchestrations?

### DISCUSSION THREAD

**Discussion by khntst** [GoogleCloudPlatform/terraformer#1505] [Ideas]
Would current Terraformer functionality for PagerDuty already be able to emit Event Orchestration type rulesets?  The PagerDuty 'official' provider was enhanced to include this support since last update in Terraformer.

I noticed this pull request to enhance other functionality:
https://github.com/GoogleCloudPlatform/terraformer/pull/1095
... but this seems to have stalled.

I wonder what it would take to accomplish Event Orchestration in TF-er.  Maybe adapt the current Ruleset generator code?

**Comment by Rohlik** [comment]
Also looking for it 🤞🏼.
